.github/workflows/ci.yaml

```diff
@@ -94,9 +94,15 @@ jobs:
         with:
           node-version: 12.x
 
+      - name: Audit vulnerabilities
+        run: npm audit
```

Comment:
Hm, should we run `npm audit` *after* `npm ci`?

Reply:
There is nothing about that in [docs](https://docs.npmjs.com/cli/audit) tough I guess we should move it after `npm ci` to be on the safe side